src/components/Notification/index.js

```diff
@@ -0,0 +1,43 @@
+import { h, Component } from 'preact';
+import PropTypes from 'prop-types';
+
+import { applyTheme } from '../ThemeProvider/';
+
+import './style.css';
+
+/**
+* Notification
+*
+*/
+class Notification extends Component {
+  propTypes = {
+    theme: PropTypes.string,
+    network: PropTypes.string,
+  };
+
+  render () {
+    const { theme, network } = this.props;
+    if (network === '') return null;
+
+    const renderNotification = () => {
+      switch (network) {
+        case 'disconnected':
+          return 'Chat Disconnected';
+        case 'reconnecting':
+          return 'Disconnected; trying to reconnect...';
+        case 'reconnected':
+          return 'Disconnected; trying to reconnect...';
+        default:
+          return '';
+      }
+    };
+
+    return (
+      <section className={`Notification__${theme}-${network}`}>
```

Comment:
nit, but what was the reason begind using a section tag instead of a span or a div?